src/dex/maverick-v2/maverick-v2.ts

```diff
@@ -27,9 +27,15 @@ import { MaverickV2EventPool } from './maverick-v2-pool';
 import { SUBGRAPH_TIMEOUT } from '../../constants';
 import { Interface } from '@ethersproject/abi';
 import MaverickV2PoolABI from '../../abi/maverick-v2/MaverickV2Pool.json';
+import MaverickV2RouterABI from '../../abi/maverick-v2/MaverickV2Router.json';
 import ERC20ABI from '../../abi/erc20.json';
 import { extractReturnAmountPosition } from '../../executor/utils';
+
 const EFFICIENCY_FACTOR = 3;
```

Comment:
Added `MIN_POOL_VOLUME_USD` and `MIN_POOL_LIQUDITY_USD` to filter out bad pools

Reply:
We now supports all Maverick's pools so there is no need in this constants anymore